cli/src/main.rs

```diff
@@ -363,6 +364,24 @@ enum Commands {
         #[arg(value_parser = clap_enum_variants!(FieldArgument))]
         field: FieldArgument,
     },
+
+    /// Executes all functions starting with `test_` in every module called
+    /// `test` starting from the given module.
+    Test {
+        /// Input file.
+        file: String,
+
+        /// The field to use
+        #[arg(long)]
+        #[arg(default_value_t = FieldArgument::Gl)]
+        #[arg(value_parser = clap_enum_variants!(FieldArgument))]
+        field: FieldArgument,
+
+        /// Also run the tests inside the standard library.
+        #[arg(long)]
+        #[arg(default_value_t = false)]
+        include_std_tests: bool,
```

Comment:
It filters tests based on their namespace. Since `std` is always included, we need to filter the tests out. If you run `powdr test std` with the filter active, it will not run any tests.

Reply:
Hmm, but does this need to leak to the CLI? Or could we just say we never want to run the std tests, unless we pass `std` explicitly?